### ISSUE TITLE
reimplement sharing a file

### ISSUE THREAD

**Issue by jmt85** [Qabel/qabel-android#562]
* [x] Send a file share #620 
* [x] Receive a file share
* [x] Show in the file browser that a file is shared
* [x] Unsharing of files
* [ ] List of incoming shares
* [ ] List of outgoing shares

**Comment by jmt85** [comment]
see #565

**Comment by audax** [comment]
Depends on #522

**Comment by JoGir** [comment]
FileBrowser refreshes content and configures toolbar onActivityCreated only, configureAsMainFragment is missing
```
onResume() ...
       setHasOptionsMenu(true)
       configureAsMainFragment()
       refresh()

**Comment by JoGir** [comment]
moved last points to closed #721
